src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.ts

```diff
@@ -66,6 +71,26 @@ export class InstructorHelpSessionsSectionComponent extends InstructorHelpSectio
     hasEmailSettingsPanelExpanded: true,
   };
 
+  readonly exampleModel: CommentEditFormModel = {
```

Comment:
and delete `readonly`.

Reply:
Updated, but there did not seem to be changes reflected on edit form.

Reply:
If we do not enabling the 2-way data binding, you will not be able to enter the comment text in the form.